extension/googleclientauthextension/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+type: googleclientauth
+
+status:
+  class: extension
+  stability:
+    development: [extension]
+  codeowners:
+    active: [dashpole, damemi]
```

Comment:
Since this component is being added to the bug/other/feature request templates, and code owners are being added here, we probably want to add an entry to the [code owners file](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS).

Reply:
does `make update-codeowners` do that? that command fails for me btw:
```
make update-codeowners
go: cannot load module exporter/f5cloudexporter listed in go.work file: open exporter/f5cloudexporter/go.mod: no such file or directory
go: cannot load module extension/httpforwarder listed in go.work file: open extension/httpforwarder/go.mod: no such file or directory
go: cannot load module exporter/f5cloudexporter listed in go.work file: open exporter/f5cloudexporter/go.mod: no such file or directory
go: cannot load module extension/httpforwarder listed in go.work file: open extension/httpforwarder/go.mod: no such file or directory
cd cmd/githubgen && go install .
go: cannot load module ../../exporter/f5cloudexporter listed in go.work file: open ../../exporter/f5cloudexporter/go.mod: no such file or directory
go: cannot load module ../../extension/httpforwarder listed in go.work file: open ../../extension/httpforwarder/go.mod: no such file or directory
make: *** [Makefile:271: githubgen-install] Error 1
```

Reply:
Deleted my last comment, that error was from my own go workspaces. With that updated, `make update-codeowners` still fails:

```
make update-codeowners
no Go files in /usr/local/google/home/mikedame/go/src/github.com/open-telemetry/opentelemetry-collector-contrib
cd cmd/githubgen && go install .
githubgen
2024/03/05 14:16:14 codeowners are not members: BinaryFissionGames, Frapschen, JonathanWamsley, MovieStoreGuy, Mrod1598, StefanKurek, VihasMakwana, adrielp, ag-ramachandran, akats7, alexvanboxel, antonblock, armstrmi, atingchen, atoulme, avadhut123pisal, bertysentry, braydonk, bryan-aguilar, chrroberts-pure, codeboten, cparkins, cpheps, damemi, dashpole, dehaansa, dgoscn, eedorenko, erichsueh3, evan-bradley, fatsheep9146, fchikwekwe, gjanco, greatestusername, hickeyma, hughesjj, jacobmarble, jamesmoessis, jmacd, jsuereth, kentquirk, khyatigandhi6, kkujawa-sumo, liustanley, mapno, mar4uk, marctc, martin-majlis-s1, matej-g, mcardy, moh-osman3, mrsillydog, nslaughter, pavankrish123, pcwiese, pjanotti, portertech, povilasv, psx95, pxaws, rapphil, rmfitzpatrick, rnishtala-sumo, rogercoll, samiura, schmikei, shalper2, sinkingpoint, smithclay, splunkericl, srprash, swiatekm-sumo, tamir-michaeli, tomaz-s1, varunraiko, wangzlei, yotamloe, zdaratom-s1, zpzhuSplunk
make: *** [Makefile:275: gengithub] Error 1
```

Reply:
Thanks for pointing this out, it's fine to update code owners manually if the command fails. It looks like that failure is due to a new user being added who is working on a new component but not yet a member of the OpenTelemetry community, nothing on your end 👍

Reply:
cool, thanks!